src/transformStateWithClones.js

```diff
@@ -6,8 +6,44 @@
  *
  * @return {Object[]}
  */
+function removeProperties(object, properties) {
+  for (const item of properties) {
+    delete object[item];
+  }
+}
+
+function clearObj(object) {
```

Comment:
Don't make the function name or variable name shortened. Use full names for better readability
```suggestion
function clearObject(object) {
```